src/app/page.tsx

```diff
@@ -184,39 +184,41 @@ export default function Home() {
           </div>
         </section>
 
-        <section className="container flex flex-col lg:min-h-[100dvh] items-center justify-center py-10 md:py-24 lg:py-20">
-          <div className="flex w-full md:w-2/3 lg:1/2 text-center md:text-left flex-col px-10">
-            <h2 className="text-3xl md:text-4xl font-bold text-center">
-              See what our users have to say
-            </h2>
-          </div>
-          <div className="flex px-10">
-            <Testimonials />
-          </div>
-        </section>
-
         <section className="container flex flex-col lg:min-h-[100dvh] items-center justify-center py-10 md:py-24 lg:py-20">
           <div className="flex w-full md:w-1/2 text-center md:text-left flex-col px-10">
             <h2 className="text-3xl md:text-4xl font-bold text-center">
               Meet some of our mentors
             </h2>
           </div>
-          <div className="flex px-10 mt-10 w-full">
+          <div className="flex px-10 my-10 w-full">
             <Carousel
               opts={{
                 align: "start",
                 loop: true,
               }}
               className="w-full flex flex-col items-center justify-start text-center"
             >
-              <CarouselContent className="w-screen">
-                <Mentors />
-              </CarouselContent>
-              <CarouselPrevious />
-              <CarouselNext />
+              <div className="overflow_mask w-screen md:w-full">
```

Comment:
What is "overflow_mask"? I don't think that's tailwind right? If we do want to create new classes can we stick with similar syntax, and use a dash (-) instead of underscore (_)

Reply:
I used the _ to be able to differentiate it from tailwind classes. So that whoever takes a look at it will know that it's not traditional tailwind. That style is defined in globals.css. But if you'd like me to change it to - instead, I can do that too. We will have to come up with another way to differentiate it though.